src/main/java/com/zaxxer/hikari/pool/HikariPool.java

```diff
@@ -526,7 +526,7 @@ private void fillPool()
          addConnectionExecutor.execute(new Runnable() {
             @Override
             public void run() {
-               logPoolState("After fill ");
+               logPoolState("After fill\t");
```

Comment:
Is that all the callers?  If so, might as well just put the \t in the ``logPoolState()`` method.